os/kernel/debug/mem_leak_checker.c

```diff
@@ -377,6 +380,11 @@ int run_mem_leak_checker(int checker_pid, char *bin_name)
 	int broken_cnt = 0;
 	struct mm_heap_s *heap = NULL;
 
+#if defined(CONFIG_ARCH_BOARD_RTL8730E)
+	/* Set reboot reason temporaily to avoid km4 log print */
+	WRITE_REBOOT_REASON(REBOOT_TEMP_BLOCK_KM4_LOG);
```

Comment:
How about make up_pause_subcore_logging()?

The KM4 is related to only rtl8730e chip.